crates/iota-sdk-ffi/src/error.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fmt;
+
+use uniffi::Error;
+
+pub type Result<T, E = BindingsSdkError> = std::result::Result<T, E>;
+
+#[derive(Debug, Error)]
+#[uniffi(flat_error)]
+pub enum BindingsSdkError {
+    Generic(String),
```

Comment:
Will we make no effort for having a more granular error? Is there a technical limitation due to uniffi?

Reply:
The error exists only because of the format that uniffi requires, i.e. it must be an enum. However, there's really no reason in my view to expand it